stable/datacube-index/templates/cronjob-index.yaml

```
@@ -29,8 +29,7 @@ spec:
             app.kubernetes.io/instance: {{ .Release.Name }}
             heritage: {{ include "datacube-index.fullname" . }}
           annotations:
-
-{{ toYaml $index.annotations | indent 8 }}
+          {{ toYaml $index.annotations | indent 8 }}
```

```suggestion
{{ toYaml $index.annotations | indent 10 }}
```
Indenting this statement and leaving the `ident 8` in will indent it quite a bit further then expected.